datasets.py

```diff
@@ -139,13 +140,18 @@ def __init__(self, transform):
         # self.scene_mapping = setup_scenemapping(self, '[name of dataset]')
         self.scene_mapping = NoneDict()
 
+        #Note: Any of the 'optional' attributes may be necessary depending on analysis and metrics, check before not filling in
+
+
         # Maps each label to number of supercategory group, which is listed in keys of GROUPINGS_TO_NAMES (optional)
         self.group_mapping = None
 
         # Labels that correspond to people (optional)
+        #Note: This refers to labels that describe people in images
         self.people_labels = []
 
         # Number of images from dataset that are female at index 0 and male at index 1 (optional, doesn't need to exist)
+        #Note: Needs to exist for gender analysis, is count for each gender image
```

Comment:
can remove, your above note on how optional is not fully optional covers this point